docs/unignore_outcomes.rst

```diff
@@ -0,0 +1,30 @@
+=================
+Unignore Outcomes
+=================
+
+By default, when using ``ignore_outcome`` in your Tox configuration,
+any build errors will show as successful on Travis. This might not
+be desired, as you might want to control allowed failures inside your
+``.travis.yml``. To cater this need, you can set ``unignore_outcomes``
+to ``True``. This will override ``ignore_outcome`` by setting it to
+``False`` for all environments.
+
+
+Usage
+=====
+
+Configure the allowed failures in the build matrix in your ``travis.yml``:
+
+.. code-block:: yaml
+
+    matrix:
+      allow_failures:
+      - python: 3.6
+        env: DJANGO=master
+
+And in your ``tox.ini``:
+
+.. code-block:: ini
+
+    [travis]
+    unignore_outcomes = True
```

Comment:
I see this as a feature of the tox automatic environment discovery, rather than it's own distinct feature. Unless you disagree, let's put this in a new section in the docs for `toxenv` instead of in its own file. As the docs get cleaned up, I forsee things being broken up further for documentation, but let's not try to do that in this PR.